packages/gatsby/src/redux/actions/public.ts

```diff
@@ -1093,7 +1121,10 @@ actions.setWebpackConfig = (config: Object, plugin?: ?Plugin = null) => {
  *
  * @param {Object} config complete webpack config
  */
-actions.replaceWebpackConfig = (config: Object, plugin?: ?Plugin = null) => {
+actions.replaceWebpackConfig = (
+  config: Record<string, any>,
```

Comment:
I think this should be something like webpack.Config imported from webpack.

Reply:
Done in 3ccf334